.circleci/dynamic_config.yml

```diff
@@ -325,6 +345,60 @@ jobs:
                 no_output_timeout: 40m
       - fail_fast
 
+  bazel-e2e-tests:
+    executor: bazel-executor
+    parallelism: 4
+    parameters:
+      build_type:
+        type: enum
+        enum: *all_e2e_build_types
+        default: 'e2e'
+      subset:
+        type: enum
+        enum: *all_e2e_subsets
+        default: 'npm'
+    steps:
+      - custom_attach_workspace
+      - browser-tools/install-chrome
```

Comment:
Why is chrome needed?

Reply:
Without it I'm getting https://app.circleci.com/pipelines/github/angular/angular-cli/26617/workflows/8a85cca7-f1fb-4d5a-8698-83e522f485d4/jobs/352420/steps?invite=true#step-106-1703

... which might just required propper config of chrome within bazel. I can look into it more. It would be great to drop this step which requires 20-30s of CI time.

Reply:
This is also used in the unit tests (bazel) and non-bazel e2e tests, so maybe it should stay?

Reply:
As I understand this it should not be necessary because we provide the browsers via bazel